lib/src/provider.dart

```diff
@@ -1,4 +1,4 @@
-enum Provider { azure, bitbucket, facebook, github, gitlab, google }
+enum Provider { apple, azure, bitbucket, discord, facebook, github, gitlab, google, twitter }
```

Comment:
Appreciate that you sorted these in alphabetical order! There seems to be a formatting error that can be fixed by replacing this line with this:

```suggestion
enum Provider {
  apple,
  azure,
  bitbucket,
  discord,
  facebook,
  github,
  gitlab,
  google,
  twitter
}
```